client/src/components/List/List.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import ToDoListItem from "../ToDoListItem/ToDoListItem";
+import styled from "@emotion/styled";
+
+const toDos = [
+  {
+    id: 1,
+    title: "Angebot anfordern",
+    category: "Location",
+    completed: false,
+  },
+  {
+    id: 2,
+    title: "Dj anrufen",
+    category: "Musik",
+    completed: true,
+  },
+  {
+    id: 3,
+    title: "Kleid auswählen",
+    category: "Bekleidung",
+    completed: false,
+  },
+  {
+    id: 4,
+    title: "Menü abstimmen",
+    category: "Location",
+    completed: false,
+  },
+];
```

Comment:
You could think of fetching these from your db.json as you already have them added.

Reply:
You are right, this will be one of the next steps.